examples/asr/hkust/configs/mtl_transformer.json

```diff
@@ -2,6 +2,8 @@
   "batch_size":32,
   "num_epochs":50,
   "sorta_epoch":1,
+  "log": "examples/asr/hkust/ckpts/mtl_transformer_ctc/nohup.out",
```

Comment:
These options should also be added in examples/asr/aishell_sub/configs/mtl_transformer.json